src/libraries/System.Drawing.Common/src/System/Drawing/Icon.Windows.cs

```diff
@@ -641,7 +641,9 @@ public void Save(Stream outputStream)
                     {
                         // We threw this way on NetFX
                         if (outputStream == null)
+#pragma warning disable CA2208 // Instantiate argument exceptions correctly
                             throw new ArgumentNullException("dataStream");
+#pragma warning restore CA2208
```

Comment:
Why not `nameof(outputStream)`?

Reply:
Because of comment above `// We threw this way on NetFX`

Reply:
Many of these shipped this way in .NET Framework.  I'm not sure why this one would be special.